presto-main/src/test/java/com/facebook/presto/operator/scalar/TestFailureFunction.java

```diff
@@ -33,13 +37,20 @@ public void testFailure()
         assertFunction("fail(json_parse('" + FAILURE_INFO + "'))", UNKNOWN, null);
     }
 
-    @Test(expectedExceptions = RuntimeException.class, expectedExceptionsMessageRegExp = "/ by zero")
```

Comment:
Can you just use `expectedExceptions = PrestoException.class, ...`?

Reply:
Using `@Test`'s `expectedExceptions`, we cannot match the error code, which is something we want to test here as well.